src/apis/DataDisplayApi.ts

```diff
@@ -0,0 +1,24 @@
+
+import axios from 'axios';
+import { DataDisplayRequest, DataDisplayResponse } from '../types/Types';
+
+const BASE_URL = 'https://0fg9aos700.execute-api.ap-south-1.amazonaws.com/asdfasdf/sample-route-key'; // Replace with your actual API base URL
+
+export const getDataDisplay = async (): Promise<DataDisplayResponse> => {
+  try {
+    const response = await axios.get<DataDisplayResponse>(`${BASE_URL}/data-display`);
```

Comment:
URL endpoint is incorrect. Use this instead: 
const response = await axios.get<DataDisplayResponse>(`${BASE_URL}`);